### ISSUE TITLE
'Already seen' banners are present when subject hasn't been seen

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#657]
**Package**
lib-classifier

**Feature or Issue Description**
Testing in VoiceOver revealed that four copies of the 'Already Seen' banners are present on new, unseen subjects.

They should only be present for seen subjects, and only one should be present for a given subject.
![Screenshot showing that VoiceOver finds four Already Seen banners on every subject.](https://user-images.githubusercontent.com/59547/55634873-4765db00-57b7-11e9-92f2-aaa99fa1f2c4.png)

**Comment by rogerhutchings** [comment]
Right - I'm going to add an animation library after all, probably `react-pose`, so I can access the `enter`/`exit` transitions. That way, the banner can be safely removed from the DOM and still animate in when active.

**Comment by eatyourgreens** [comment]
👍 I did some more playing around in VoiceOver, by the way, and I could click the hidden buttons via the controls in the rotor, although nothing happened when they were clicked.
